root/user/collections.tt

```diff
@@ -72,3 +88,47 @@
 [% END %]
 
 [% END %]
+
+[% BLOCK 'collaborator-collections' %]
+[%- IF collaborator_collections.size -%]
+    [%- FOR collabentity = collaborator_collections.pairs -%]
+        [%- IF collabentity.value.size -%]
+            <h3>[% format_entity_collection_title(collabentity.key) %]</h3>
+
+            <table class="tbl">
+                <thead>
+                    <tr>
+                        <th>[% l('Collection') %]</th>
+                        <th>[% l('Type') %]</th>
+                        <th>[% format_plural_entity_type_name(collabentity.key) %]</th>
+                        [% IF viewing_own_profile %]
```

Comment:
Any reason we don't have a column for the collaborator count down here too?

Reply:
Not really, actually. Also made a change so that it specifies if it's your collection in the privacy column (there because then it's easier to understand why you're getting shown a private collection).